src/app/options.h

```diff
@@ -1,5 +1,7 @@
 /*
- * Bittorrent Client using Qt4 and libtorrent.
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2016  Eugene Shalygin
```

Comment:
Sure. Whatever you want. It was just just a reminder.
(I am continuing with the review)